apps/cow-fi/util/getPriceChangeColor.ts

```
@@ -1,8 +1,10 @@
-import { Color } from 'styles/variables'
+import type { CowProtocolTheme } from 'styled-components'
```

ESLint

[error] 'styled-components' import is restricted from being used. Please import from styled-components/macro.

(no-restricted-imports)